.env.example

```
@@ -32,6 +32,25 @@ VITE_OPENAI_TEMPERATURE=0.1
 VITE_HALLUCIFIX_API_URL=https://api.hallucifix.com
 
+# =============================================================================
+# AWS SERVICES (Required for AWS infrastructure)
+# =============================================================================
+
+# AWS Region
+VITE_AWS_REGION=us-east-1
+
+# AWS Cognito Configuration
+VITE_COGNITO_USER_POOL_ID=us-east-1_XXXXXXXXX
+VITE_COGNITO_USER_POOL_CLIENT_ID=your_cognito_client_id
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_COGNITO_USER_POOL_CLIENT_ID key should go before the VITE_COGNITO_USER_POOL_ID key

(UnorderedKey)